src/referencePackages/src/microsoft.build.utilities.core/16.6.0/microsoft.build.utilities.core.nuspec

```diff
@@ -0,0 +1,36 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2013/05/nuspec.xsd">
+  <metadata>
+    <id>Microsoft.Build.Utilities.Core</id>
+    <version>16.6.0</version>
+    <authors>Microsoft</authors>
+    <owners>Microsoft</owners>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <license type="expression">MIT</license>
+    <licenseUrl>https://licenses.nuget.org/MIT</licenseUrl>
+    <icon>MSBuild-NuGet-Icon.png</icon>
```

Comment:
Share the pngs like the keys rather than duplicating them? (Also share with targeting packs, which already have a few duplicates?)

Reply:
I'm not sure it makes sense to do that in the pre-arcade branch because it will require additional infra to copy icon files to the output dirs that we use to pack.  It doesn't seem worth it for a handful of png files.  I think this would make sense in the master branch since the build is updated there.

Reply:
That's fair. :+1: